lib/jxl/enc_frame.cc

```diff
@@ -978,8 +975,10 @@ Status ComputeJPEGTranscodingData(const jpeg::JPEGData& jpeg_data,
           } else {
             idc = inputjpeg[base] + 1024 / qt_dc[c];
           }
-          dc_counts[c][std::min(static_cast<uint32_t>(idc + 1024),
-                                static_cast<uint32_t>(2047))]++;
+          if (c == 1) {
+            dc_counts[std::min(static_cast<uint32_t>(idc + 1024),
```

Comment:
While on this: `std::min<uint32_t>` looks neater than double `static_cast`.

Reply:
updated